tests/fixtures/addon/kitchen-sink-mu/index.js

```diff
@@ -0,0 +1,9 @@
+module.exports = {
+  name: 'basic-thing',
```

Comment:
@mixonic I wonder if these tests are failing because the name doesn't match the folder name. In the other examples they do. 
PS how do I run just these tests locally, and even better is it possible to run them in the browser?